### ISSUE TITLE
Accessibility: Dark Themes override text selection mouse cursor

### ISSUE THREAD

**Issue by stroborobo** [microsoft/vscode#140490]
Issue Type: <b>Bug</b>

1. Change your mouse cursor using the system's accessibility settings. E.g. make it pink and larger.
2. Switch VS Code to a dark theme
3. Move mouse cursor into a text editor pane or debug console.
4. Cursor is displayed as default black/white selection cursor, not your colored one.

Observed on macOS 12.1  
Does not happen in the terminal pane

VS Code version: Code 1.63.2 (899d46d82c4c95423fb7e10e68eba52050e30ba3, 2021-12-15T09:37:28.172Z)
OS version: Darwin x64 21.2.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 3, 3|
|Memory (System)|32.00GB (2.03GB free)|
|Process Argv|--crash-reporter-id 21bc40f9-2216-480b-b47e-abc548bd2c9c|
|Screen Reader|no|
|VM|0%|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
vscop804cf:30404767
vs360:30404995
vsrem710cf:30416617

```

</details>

**Comment by aeschli** [comment]
VS Code doesn't inherit the OS system colors. Instead, we have our own color themes. These work across operating systems and browsers.
There's a lot of themes to pick from the marketplace, including high contrast themes. Use the command `Preferences: Color Theme` to select or install a theme.

You can also customize a theme in the settings. For the cursor color you can do this:
```
{
"editorCursor.foreground": "#ff00FF"
}
```

Closing, no changes planed.

**Comment by stroborobo** [comment]
@aeschli  You misunderstood. I'm talking about the *mouse cursor*

**Comment by TylerLeonhardt** [comment]
![image](https://user-images.githubusercontent.com/2644648/151600702-2307c5e5-0ea7-45b8-a76c-458f8b233b9a.png)

Pink and large

**Comment by stroborobo** [comment]
Thank you!! <3
